tests/10-deploy.py

```diff
@@ -10,7 +11,9 @@ def setUpClass(cls):
         cls.d = amulet.Deployment(series='xenial')
         cls.d.add('etcd')
         cls.d.setup(timeout=1200)
-        cls.d.sentry.wait()
+        cls.d.sentry.wait_for_messages({'etcd':
+                                        re.compile('Healthy*|Unhealthy*')})
```

Comment:
I have not used this method before. It will take a regular expression? Thanks @johnsca !